src/palladio/NodeSpareParameter.cpp

```diff
@@ -86,42 +87,131 @@ void addParmsFromTemplateArray(OP_Node* node, PRM_Template* spareParmTemplates,
 }
 
 void addParm(OP_Node* node, PRM_Type parmType, const std::wstring& id, const std::wstring& name, PRM_Default defaultVal,
-             PRM_Range* range, const FolderVec& parentFolders) {
+             PRM_Range* range, const FolderVec& parentFolders, const std::wstring& annotation) {

```

Comment:
consider renaming `annotation` to `description` for clarity

Reply:
done